app/views/service_specific_information/index.html.erb

```diff
@@ -13,12 +13,12 @@
         <li><a href="#army" id="tab-army"><span>U.S. </span>Army</a></li>
         <li><a href="#air-force" id="tab-air-force"><span>U.S. </span>Air Force</a></li>
         <li><a href="#navy" id="tab-navy"><span>U.S. </span>Navy</a></li>
-        <li><a href="#marines" id="tab-marine-corps"><span>U.S. </span>Marine Corps</a></li>
+        <li><a href="#marines" id="tab-marines"><span>U.S. </span>Marine Corps</a></li>
         <li><a href="#coast-guard" id="tab-coast-guard"><span>U.S. </span>Coast Guard</a></li>
       </ul>
 
       <div class="usa-grid-full tabs-panel" id="army">
-        <%= image_tag("Service-Army.png") %><br>
+        <%= image_tag("Service-Army.png", :class => "svc-spec-banner") %><br>
```

Comment:
Suggest using the newer Ruby 1.9 hash syntax here:

```erb
<%= image_tag 'Service-Army.png', class: 'svc-spec-banner' %>
```

The main difference being use of `foo: 'bar'`-style syntax vs. `:foo => 'bar'`.